Visi.Repository/PatientQueryFactory.cs

```diff
@@ -50,6 +50,19 @@ public override PatientQuery AddValueFilter(string parameterName, ReferenceFromV
             return base.AddValueFilter(parameterName, value);
         }
 
+        public PatientQuery AddValueFilter(string parameterName, MissingValue value)
+        {
+            var isMissing = value.IsMissing; // true
+
+            switch (parameterName)
+            {
+                case "_id": return PredicateQuery(p => (p.Id == null) == isMissing);
```

Comment:
I would not include an example on `_id`. As that is a mandatory element on any resource, it is a confusing example.